packages/framework/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@novu/echo",
-  "version": "0.24.3-alpha.4",
+  "name": "@novu/framework",
+  "version": "0.24.3-alpha.5",
```

Comment:
updated alpha version here

Reply:
Also, the author is set as "Athena", we wan't to change that to "Novu" maybe?

Reply:
Yes, let's make it `Novu`